.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
+eclipse.preferences.version=1
```

Comment:
I think some of these folders/files can go into .gitignore; if you're using the eclipse IDE, you can find a specific gitignore for it online